x-pack/plugins/cross_cluster_replication/public/app/sections/home/auto_follow_pattern_list/components/detail_panel/detail_panel.js

```diff
@@ -251,16 +248,16 @@ export class DetailPanelUi extends Component {
 
   renderFooter() {
     const {
-      editAutoFollowPattern,
       autoFollowPattern,
-      autoFollowPatternName,
       closeDetailPanel,
     } = this.props;
 
     if (!autoFollowPattern) {
       return null;
     }
 
+    const { name: autoFollowPatternName } = autoFollowPattern;
```

Comment:
If the auto-follow pattern doesn't exist, then the name will be undefined. Compare this to Remote Clusters, which will still show you the name if the remote cluster can't be found.

![image](https://user-images.githubusercontent.com/1238659/50249265-5785b600-0392-11e9-9fcb-f8461bff8b5b.png)

![image](https://user-images.githubusercontent.com/1238659/50249271-5a80a680-0392-11e9-9953-ea5e30657e64.png)

Reply:
Good catch